solution.txt

```diff
@@ -0,0 +1,260 @@
+/*
+Exercise 1:
+  - Define an empty array named foods
+*/
+
+// Exercise 1 has been completed for you...
+
+const foods = [];
+
+console.log('Exercise 1 Result:\n', foods);
+
+/*
+Exercise 2:
+  - Add the strings 'pizza' & 'cheeseburger' to the foods array such that 'pizza' comes before 'cheeseburger'.
+*/
+
+// Complete Exercise 2 below...
+
+foods.push('pizza', 'cheeseburger');
+
+console.log('Exercise 2 Result:\n',  foods);
+
+/*
+Exercise 3:
+  - Add the string 'taco' to the foods array so that 'taco' is the first food in the array.
+*/
+
+// Complete Exercise 3 below...
+
+foods.unshift('tacos');
+
+console.log('Exercise 3 Result:\n', foods);
+
+/*
+Exercise 4:
+  - Access the string 'pizza' (based upon its known position) in the foods array and assign to a variable named favFood.
+*/
+
+// Complete Exercise 4 below...
+
+//var favFood = foods.indexOf('pizza');
+
+var favFood = foods.slice(foods.indexOf('pizza'),foods.indexOf('pizza') + 1);
+favFood = favFood.toString();
```

Comment:
nice! I'm curious if you can solve this prompt using `splice`, just to solidify the difference between the two? Also, if you do this, it's worth keeping in mind that `slice` returns a copy of the original array, whereas `splice` alters the original array. Definitely not required, but might be an interesting thing for you to do